### DISCUSSION TITLE
Public meeting list not displaying!

### DISCUSSION THREAD

**Discussion by Spsrky** [code4recovery/12-step-meeting-list#862] [Q&A] [answered]
URGENT! We were having trouble adding a new meeting so I deleted the plugin and reinstalled, then uploaded the saved CSV. Meetings now are showing in the backend but there is nothing on the public side meetings list at https://www.pdxaa.org/index.php/meetings/. During upload there was a Google Maps error message for every meeting saying we needed to enable billing, but we are using Mapbox and have the Mapbox key entered in Settings.

**Comment by kiyote33** [comment, marked as answer]
Hi @Spsrky, sorry to hear about your meeting list problems.  Quite often when redoing the 12 Step Meeting List plugin, it's necessary to reset the WordPress permalinks. Go to the Settings/Permalinks page on your admin menu and simply click on the Save Changes button, which should get meetings displaying again on your /meetings page.  Let me know if that doesn't resolve your issue and I'll dig a little deeper into your error messages.

**Comment by Spsrky** [comment]
Hi Wayne,

Brilliant - that worked! Boy, that never would have occurred to me - thanks!

The original problem with entering a new meeting still persists but I will start a new discussion thread for that and mark this one closed.

In fellowship and service,

Theron
